app/policies/observation_policy.rb

```diff
@@ -0,0 +1,9 @@
+class ObservationPolicy
+  include ExceptionPolicy
+
+  def can_destroy!
+    role = Role.new(@user, @record.proposal)
+    check(@user.id == @record.user_id || role.client_admin? || @user.admin?,
+          "Only the observer can delete themself")
```

Comment:
Hmm, I would think that anyone who can edit the request should be able to delete the observation, in case they accidentally add the wrong person or something.